changelog/update-4144-remove-the-color-fields-on-receipt-settings

```diff
@@ -0,0 +1,4 @@
+Significance: minor
+Type: update
+
+Remove Stripe related branding options from the card readers settigs page.
```

Comment:
There's a small typo here in "settings." Also, "Stripe-specific" would be more appropriate than "Stripe-related."

Reply:
That's a nice precision thanks